exercises/simple-linked-list/src/example.erl

```diff
@@ -0,0 +1,33 @@
+-module(example).
+
+-export([cons/2, count/1, empty/0, from_array/1, head/1,
+	 reverse/1, tail/1, to_array/1]).
+
+empty() -> empty.
+
+cons(Elt, List) -> {Elt, List}.
+
+head({Head, _Tail}) -> Head.
+
+tail({_Head, Tail}) -> Tail.
+
+reverse(List) -> reverse(List, empty).
+
+reverse(empty, Acc) -> Acc;
+reverse({Head, Tail}, Acc) ->
+    reverse(Tail, cons(Head, Acc)).
+
+count(empty) -> 0;
+count({_Head, Tail}) -> 1 + count(Tail).
+
+to_array(List) ->
```

Comment:
I'd prefer to have `to_native_list/1` and `from_native_list/1`, usage of `array` is not very common in erlang, and in your tests, you are creating the `array` from a (native) list anyway…

Reply:
yes, that was akward